modules/machineset-creating.adoc

```diff
@@ -6,13 +6,15 @@
 = Creating a MachineSet
 
 You can create more MachineSets. Because the MachineSet definition contains
-details that are specific to the AWS region that the cluster is deployed in,
-you copy an existing MachineSet from your cluster and modify it.
+details that are specific to the Amazon Web Services (AWS) region that the
+cluster is deployed in, you copy an existing MachineSet from your cluster and
+modify it.
```

Comment:
Out of scope for UPI on AWS.  This should be a separate PR.

Reply:
Depending on how we land with the "pull MachineSets from UPI" or "fix MachineSets after UPI install" debate, this might be in scope.